assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -32,17 +37,42 @@ import {
 	ProfileNameTextField,
 	ExistingGTMPropertyNotice,
 } from '../common';
+import {
+	PropertySelect as PropertySelect4,
+} from '../../../analytics-4/components/common';
+import { isFeatureEnabled } from '../../../../features';
 import StoreErrorNotices from '../../../../components/StoreErrorNotices';
 import GA4Notice from '../common/GA4Notice';
-import { STORE_NAME } from '../../datastore/constants';
-const { useSelect } = Data;
+import {
+	SETUP_FLOW_MODE_GA4_TRANSITIONAL,
+	SETUP_FLOW_MODE_LEGACY,
+	SETUP_FLOW_MODE_UA,
+	STORE_NAME,
+} from '../../datastore/constants';
+import { MODULES_ANALYTICS_4, PROPERTY_CREATE } from '../../../analytics-4/datastore/constants';
+import SettingsNotice, { TYPE_INFO } from '../../../../components/SettingsNotice';
+import Link from '../../../../components/Link';
+const { useSelect, useDispatch } = Data;
 
 export default function SettingsForm() {
+	const ga4Enabled = isFeatureEnabled( 'ga4setup' );
```

Comment:
Let's use `useFeature` hook instead:
```suggestion
	const ga4Enabled = useFeature( 'ga4setup' );
```